### DISCUSSION TITLE
new 3 control plane setup with vmware workstation

### DISCUSSION THREAD

**Discussion by matjazantloga** [siderolabs/talos#9182] [Q&A] [answered]
Hi

Besides talos cluster on physical HPE i'm also testing it inside VMware workstation.
i have 3 control plane nodes prepared and made it so far

ma@localhost vmamd1]$ talosctl config contexts
CURRENT   NAME            ENDPOINTS                                   NODES
         TalosThinkPAD   10.99.100.132,10.99.100.133,10.99.100.134   10.99.100.132

when i try to get members i got this

[ma@localhost vmamd1]$ talosctl get members -n 10.99.100.132
rpc error: code = Unavailable desc = last connection error: connection error: desc = "transport: authentication handshake failed: tls: failed to verify certificate: x509: cannot validate certificate for 10.99.100.134 because it doesn't contain any IP SANs"

all three control planes are in this state

![image](https://github.com/user-attachments/assets/05337f6e-fa03-48a5-b226-59b8e34a2362)

I believe i'm missing sth with certificates, any idea?

**Comment by smira** [comment, marked as answer]
Yes, this is a [conflict on the host IPs and Kubernetes pod/service CIDRs](https://www.talos.dev/v1.7/introduction/troubleshooting/#conflict-on-kubernetes-and-host-subnets).

You need to change one or another so that they don't overlap.

**Comment by matjazantloga** [comment]
this is ok now. got another issue though :)

ma@localhost vmamd1]$ talosctl get members -n 10.99.100.132
rpc error: code = Unavailable desc = last connection error: connection error: desc = "transport: Error while dialing: dial tcp: lookup nodes on 10.99.202.15:53: server misbehaving"

nodes are using external dns, the machine from which i'm configuring cluster some internal DNS, not it seems like i should also do some configuration on dns, reading manual again

**Reply by smira** [reply]
Please open separate discussions for separate issues, so that they are better discoverable via the search.

In this case it seems to be your DNS server not working properly (?). I don't know what `10.99.202.15:53` is in your environment.

**Comment by matjazantloga** [comment]
on machine I'm running talosctl i've changed DNS from 10.99.100.15 to 1.1.1.1

now i'm getting

rpc error: code = Unavailable desc = last connection error: connection error: desc = "transport: Error while dialing: dial tcp: lookup nodes on 1.1.1.1:53: no such host"

it seems i should use internal and setup some zone or ?
just need direction what to do, will open new case if another issue besides DNS will arrise.



Best regards, s prijaznimi pozdravi

Matjaž Antloga, Ortoscale d.o.o.

________________________________

**Comment by matjazantloga** [comment]
i will open new one. ok.
